tasks/gc/pipeline_meta_gc.go

```diff
@@ -149,5 +152,24 @@ func (s *PipelineGC) cleanupUpgrade() error {
 	return nil
 }
 
+func (s *PipelineGC) cleanupUnseal() error {
+	// Remove sectors_unseal_pipeline entries where:
```

Comment:
This cleans up unseal records that don't have any pending steps?

Reply:
Correct, entries that have had all the work done already